plugins/sigma.neo4j.bolt/sigma.neo4j.bolt.js

```diff
@@ -0,0 +1,248 @@
+;(function (undefined) {
+    'use strict';
+
+    if (typeof sigma === 'undefined') {
+    	throw 'sigma is not declared';
+    }
+    
+    if (!('Promise' in window)) {
+    	throw 'Your browser must support Promise Object';
+    }
+    
+    var driver = null,
+    	sig = null,
+        version = null;
+    
+    // Declare neo4j package
+    sigma.utils.pkg("sigma.neo4j");
+    
+    /**
+     * This function checks if a node is already prensent in the graph
+     *
+     * @return {boolean} True if node is present
+     */
+    sigma.classes.graph.addMethod('isNodePresent', function(nodeId) {
```

Comment:
You can also just do `s.graph.nodes(id)` and it will return undefined if the node doesn't exist. This way you will avoid extending the graph object.